Tests/ImageTintingTests.swift

```diff
@@ -85,4 +85,13 @@ class ImageTintingTests: XCTestCase {
         BONAssertEqualImages(untintedResult!, tintAttemptResult!)
     }
 
+    func testAccessibilityLabelTransferToTintedImage() {
+        let accessibilityLabel = "I’m the very model of a modern accessible image."
+        let accessibleImage = UIImage(cgImage: imageForTest.cgImage!)
```

Comment:
No problem! It's pretty rare for a project like this. I should make sure the readme includes enough info on contributing. I never imagined we'd get as much community involvement as we have!